doc/api/util.md

```diff
@@ -10,6 +10,67 @@ module developers as well. It can be accessed using:
 const util = require('util');
 ```
 
+## util.callbackify(original)
+<!-- YAML
+added: REPLACEME
+-->
+
+* `original` {Function} An `async` function
+* Returns: {Function} a callback style function
+
+This method converts Promise returning API endpoints to ones that use callbacks.
```

Comment:
“API endpoints”? It takes functions, independent of context … I realize that would be a bit of duplication with the next sentence, but I’d rather have duplication than confusion (also, I’m quite sure it’s `Promise-returning` if you use it as an attribute)